### PR TITLE
Rh qudt shacl validation

### PR DESCRIPTION
**Pull request by ralphtq**

### CODE REVIEW

**Comment by steveraysteveray** [comment]
Ralph,
While looking this over, I noticed that all (I didn't check for all) the restrictions now contain:
      rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;

I'm thinking this might have been caused when you were adding that to normal class definitions. Seems a bit excessive for an anonymous class restriction. Could you remove those?

File: schema/SCHEMA_QUDT-v2.1.ttl [hunk 1]
```diff
@@ -1047,24 +1047,12 @@ qudt:Quantity
       owl:allValuesFrom qudt:QuantityValue ;
       owl:onProperty qudt:quantityValue ;
     ] ;
-  rdfs:subClassOf [
```
Comment (steveraysteveray) [inline schema/SCHEMA_QUDT-v2.1.ttl]: We still want Quantity to point to QuantityKind.
We don't want QuantityValue to point to QuantityKind.

Comment (ralphtq) [reply schema/SCHEMA_QUDT-v2.1.ttl]: This is done

**Comment by ralphtq** [comment]
@steveraysteveray  - are we done now on this PR?

File: schema/SCHEMA_QUDT-v2.1.ttl (hunk 1 above)
Comment (steveraysteveray) [reply schema/SCHEMA_QUDT-v2.1.ttl]: I'm not seeing Quantity pointing to QuantityKind.

**Comment by steveraysteveray** [comment]
I don't think so. Quantity needs to point to QuantityKind.

**Pull request merged**